File: pythonstudy_day12/appliance/TV.py
from appliance.MediaAppliance import MediaAppliance

class TV(MediaAppliance):
    def __init__(self,power,volumn,size):
        super().__init__(power,volumn)
        self.size = size
    def watch(self):
        str = "hi have fun" if self.power else "switch on"
        print(str)

File: day12/src/com/inheritance/TV.java
package com.inheritance;
import java.io.*;
import java.util.*;

/**
 * 
 */
public class TV extends Media {
	private int size;

    /**
     * Default constructor
     */
    public TV() {
    	super();
    }
    
    public TV(boolean power, int volumn, int size) {
    	super(power, volumn);
    	this.size = size;
    }


     
    public int getSize() {
		return size;
	}

	public void setSize(int size) {
		this.size = size;
	}

	public void watch() {
        // TODO implement here
		String str = (this.isPower()) ? "hi have fun":"switch on";
    	System.out.println(str);
    }

}